app/controllers/users_controller.rb

```diff
@@ -4,34 +4,54 @@ class UsersController < ApplicationController
   before_action :require_sign_in
   load_and_authorize_resource
   load_and_authorize_resource :vita_partner, collection: [:edit, :update], parent: false
-
+  before_action :authorize_vita_partner_updates, only: :update
```

Comment:
`authorize_vita_partner_updates` is called twice: as a before action and inside of update